account_reconciliation_widget/models/account_bank_statement.py

```diff
@@ -271,6 +272,12 @@ def _create_counterpart_and_new_aml(
 
         return counterpart_moves
 
+    def _update_move_ref(self, move_ref):
```

Comment:
As this is returning a value, not setting it, it should be `_get_move_ref`

Reply:
done